cmd/modern/root/start.go

```diff
@@ -33,24 +34,24 @@ func (c *Start) run() {
 
 	if config.CurrentContextName() == "" {
 		output.FatalfWithHintExamples([][]string{
-			{"To view available contexts", "sqlcmd config get-contexts"},
-		}, "No current context")
+			{localizer.Sprintf("To view available contexts", "sqlcmd config get-contexts")},
```

Comment:
I don't think this is right, you need two calls for Sprintf here

Reply:
Updated the Sprintf call. Not using localizer Sprintf for sqlcmd command text